Source/WTF/wtf/StdLibExtras.h

```diff
@@ -1070,13 +1070,13 @@ template<typename T, std::size_t TExtent, typename U, std::size_t UExtent>
     requires(TriviallyComparableOneByteCodeUnits<T, U>)
 size_t find(std::span<T, TExtent> haystack, std::span<U, UExtent> needle)
 {
-#if !HAVE(MEMMEM)
     if (needle.empty())
         return 0;
 
     if (haystack.size() < needle.size())
         return notFound;
```

Comment:
Why is this check needed? This is already what `memem` does when the needle size is greater than the haystack size.